src/components/atoms/ExpandIndicator/ExpandIndicator.tsx

```diff
@@ -10,7 +10,7 @@ export const ExpandIndicator = ({
 }) => {
     const className =
         variant === 'solid'
-            ? 'h-3.5 w-3.5 laptop:h-5 laptop:w-5 text-white'
+            ? 'h-3.5 w-3.5 tablet:h-4 tablet:w-4 laptop:h-5 laptop:w-5 text-neutral-400'
```

Comment:
Changed style according to https://www.figma.com/design/o2wjskNCdkt65haAgWPxo6/Responsive-Designs-(Post-Mainnet-Launch)?node-id=15840-48187&m=dev